model/src/pyrenew/model/hospitalizations.py

```diff
@@ -87,10 +88,13 @@ def __init__(
             gen_int=gen_int,
             I0=I0,
             latent_infections=latent_infections,
-            observation_process=None,
+            observation_process=NullObservation(),
```

Comment:
Equivalent but clearer to pass `None` here as well, I think.

Or could allow for an infection observation process (default `None`) to be specified at `HospitalizationsModel` instantiation?

Reply:
In principle, we could pass observed infections and thus use `observation_process` for the `RtInfectionsModel` within the `HospitalizationsModel`. Do you think we should?

Reply:
OK, 6047e60 now passes infections during construction (observation process) and when sampling. Both optionals. This would be the first multisignal model :P.